eq/client/glx/pipe.cpp

```diff
@@ -261,6 +260,10 @@ bool Pipe::_configInitGLXEW()
     {
         LBVERB << "Pipe GLXEW initialization successful" << std::endl;
         success = configInitGL();
+
+        const char* glVersion = (const char*)glGetString( GL_VERSION );
```

Comment:
Don't understand: configInitGL might have been unsuccessful, in which glGet should not be called.